lib/modules/coverage.js

```diff
@@ -330,7 +331,13 @@ internals.instrument = function (filename) {
 
     // Parse tree
 
-    const tree = BabelESLint.parse(content, Eslintrc.parserOptions);
+    const eslintConfig = internals.EslintEngine.getConfigForFile(filename);
```

Comment:
There is an error with the coverage feature regarding the `lib/linter/.eslintrc.js` file. You can find the error in the CI log: https://github.com/hapijs/lab/pull/1013/checks?check_run_id=2868449419. I can't seem to find a solution. FWIU the problem comes from this line but it seems to work properly for the rest of the `lab` codebase but not for the `.eslintrc.js` file. Perhaps I miss configured `eslint`. If anyone has a clue here, I'm all up for it.

Reply:
@nlf found the root of the problem 🙏 :
![image](https://user-images.githubusercontent.com/583657/123461023-cb028100-d5e8-11eb-97fd-dad3fc841f5a.png)